components/Form/Form.js

```diff
@@ -0,0 +1,185 @@
+import { useState } from "react";
+import {
+  Form,
+  Fieldset,
+  Legend,
+  Label,
+  Input,
+  Textarea,
+  Select,
+  Checkbox,
+  FormButton,
+  SomeoneElse,
+} from "./Form.styled";
+import PlanList from "../PlanList/PlanList";
+export default function Plan() {
+  const [plan, setPlan] = useState([]);
+  const [forSomeoneElse, setForSomeoneElse] = useState(false);
```

Comment:
Isn't this information part of the `plan` object? Why are you using a separate state variable?

Reply:
because the forSomeoneElse state variable is not directly related to the medication plan itself, but rather to the user's interaction with the form. The state variable is used to determine whether or not to display additional inputs, which is a separate concern from managing the medication plan data.

Reply:
This makes sense to me after I tried it out!